app/controllers/v0/decision_review_evidences_controller.rb

```diff
@@ -39,12 +28,26 @@ def save_attachment_to_cloud! # rubocop:disable Metrics/MethodLength
         super
       end
 
-      log_formatted(**common_log_params.merge(is_success: true))
+      params = {
```

Comment:
nit picky, but can we rename this to `log_params`? — when i see `params` in a controller i think of the [`params` hash from the request](https://guides.rubyonrails.org/action_controller_overview.html#parameters)